src/flownet/config_parser/_config_parser_pred.py

```diff
@@ -21,6 +21,33 @@ def create_schema(_to_abs_path) -> Dict:
         Dictionary to be used as configsuite type schema
 
     """
+
+    @configsuite.transformation_msg("Tries to convert input to absolute path")
```

Comment:
These are the same functions as in the other config parser file. Instead of duplicating the code we should make them imports in the both config parser files.

Reply:
I have moved all transformation methods to a new script: _config_transformation.py, except for the _to_abs_path method. This method needs the config_folder as an additional argument and I have not found a way to pass this to a transformer method.